feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsUiState.kt

```diff
@@ -206,6 +209,15 @@ sealed interface MaxVideoDurationUiState {
     ) : MaxVideoDurationUiState
 }
 
+sealed interface VideoQualityUiState {
+    data class Enabled(
+        val currentVideoQuality: VideoQuality,
+        val availableVideoQualities: List<VideoQuality>
```

Comment:
The pattern we have been using for other settings is to explicitly list out all of the options and use `SingleSelectableState` for each of them. If the device doesn't support one of the options, it will still show up in the list, but will be disabled. See FpsUiState or StabilizationUiState as examples.